specs/packagemanager/PackageManagement.md

```diff
@@ -344,6 +347,12 @@ package management APIs in Windows (e.g. Windows.Management.Deployment.PackageMa
   not PackageFamilyName. `PackageDeploymentManager` provides a richer API accepting additional
```

Comment:
Typo on line 339: `RemovePackageByFullNameAsyn` should be `RemovePackageByFullNameAsyn`.

Reply:
Lolz! The typo fix has a typo :P

RECOMMEND: Fix the typo - Async, not Asyn